### DISCUSSION TITLE
Calcul TEMPO

### DISCUSSION THREAD

**Discussion by grrosminet** [MyElectricalData/myelectricaldata_import#460] [General]
Bonjour,

Juste une petite question pour m'assurer de ce qui est affiché dans le bloc "Comparatif abonnement"

Aujourd'hui, je suis configuré en HP/HC avec 2 tranches d'HC (02h04-07h04 + 14h04-17h04)
Je me pose la question suivante : dans le calcul du tempo, est-ce que le calcul est fait en prenant mes horaires d'HC actuelles ou en prenant d'office 22h-6h ? (qui, a priori, ne sont pas négociables)

Merci et bonne journée :-)

**Comment by m4dm4rtig4n** [comment]
Non le calcul tempo est forcé sur les horaires 22h - 6h
